hangman.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'random-word'
+
+class Hangman
+  @word = ''
+  @guessedChars = []
+
+  def initialize
+    @word = RandomWord.nouns.next
+    @unmaskedWord = mask
+    @attempts = 6
+  end
+
+  def word
+    @word
```

Comment:
Ruby developers are lazy and therefore have implemented some nice ways of defining methods like this.
Specifically, `attr_reader`, `attr_writer` and `attr_accessor`.

See: https://mixandgo.com/learn/ruby_attr_accessor_attr_reader_attr_writer